pkg/smokescreen/conntrack/instrumented_conn.go

```diff
@@ -112,18 +111,16 @@ func (ic *InstrumentedConn) Close() error {
 		}
 	}
 
-	ic.tracker.Log.WithFields(logrus.Fields{
-		"idle":        idle,
-		"bytes_in":    ic.BytesIn,
-		"bytes_out":   ic.BytesOut,
-		"role":        ic.Role,
-		"req_host":    ic.OutboundHost,
-		"remote_addr": ic.Conn.RemoteAddr(),
-		"start_time":  ic.Start.UTC(),
-		"end_time":    end.UTC(),
-		"duration":    duration,
-		"timed_out":   timeout,
-		"error":       errorMessage,
+	ic.logger.WithFields(logrus.Fields{
+		"bytes_in":      ic.BytesIn,
+		"bytes_out":     ic.BytesOut,
+		"role":          ic.Role,
+		"remote_addr":   ic.Conn.RemoteAddr(),
```

Comment:
Can this be changed to match the `dst_ip` / `dst_port` pair in the decision log?